rest/taskana-rest-spring/src/main/java/pro/taskana/rest/AccessIdController.java

```diff
@@ -66,4 +75,18 @@ public static void setLdapCache(LdapCache cache) {
         ldapCache = cache;
     }
 
+    private List<AccessIdResource> validateAccessId(String accessId) throws InvalidArgumentException {
+        List<AccessIdResource> accessIdUsers;
+        if (ldapClient.useLdap()) {
+            accessIdUsers = ldapClient.searchUsersAndGroups(accessId);
+        } else {
+            accessIdUsers = ldapCache.validateAccessId(accessId);
+        }
+
+        if (accessIdUsers.size() != 1) {
```

Comment:
... but why do we check for the size 1 if the return type is a list?

Reply:
I check if the size is just 1 to validate the accessId. Now is more logical because the method just check the validation